python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -339,6 +340,29 @@ async def setup_laser(
         await self.change_laser_optical_configuration(optical_configuration)
         await self.change_laser_wavelength(wavelength, use_projector)
 
+    async def get_projector_setup(self) -> tuple:
+        """Get configuration of flatfield projector
+
+        Returns
+        -------
+            list : configuration details
+        """
+
+        select_location = await self.linearstage_projector_select.position
```

Comment:
this is awaiting on `position` from `self.linearstage_projector_select`. Is this really an await-able?

Reply:
No, it was written correctly to get the telemetry. Going to use tel_position.next(flush=True) as you recommended before